src/qservkafka/storage/votable.py

```diff
@@ -172,7 +173,12 @@ def _encode_char_column(
         bytes
             Serialized representation of the column.
         """
-        value_str = "" if value_raw is None else str(value_raw)
+        if isinstance(value_raw, datetime):
+            value_str = value_raw.strftime("%Y-%m-%dT%H:%M:%S")
```

Comment:
This is perfectly fine, just thinking out loud: if we are worried about squeezing the most performance out of the serialization should we consider using f-strings here (less readable but I think faster?):
```
if isinstance(value_raw, datetime):
    milliseconds = value_raw.microsecond // 1000
    value_str = f"{value_raw.year:04d}-{value_raw.month:02d}-{value_raw.day:02d}T{value_raw.hour:02d}:{value_raw.minute:02d}:{value_raw.second:02d}.{milliseconds:03d}" 
```

Reply:
Indeed, this is slightly faster:
```
❯ python -m timeit -s 'from datetime import datetime, UTC; value_raw = datetime.now(tz=UTC)' 'value_raw.strftime("%Y-%m-%dT%H:%M:%S") + (f".{value_raw.microsecond // 1000:03d}" if value_raw.microsecond else "")'
100000 loops, best of 5: 2.16 usec per loop
❯ python -m timeit -s 'from datetime import datetime, UTC; value_raw = datetime.now(tz=UTC)' 'milliseconds = value_raw.microsecond // 1000; f"{value_raw.year:04d}-{value_raw.month:02d}-{value_raw.day:02d}T{value_raw.hour:02d}:{value_raw.minute:02d}:{value_raw.second:02d}.{milliseconds:03d}"'
200000 loops, best of 5: 1.64 usec per loop
```
Thanks!